src/GitHub.Exports/GitHub.Exports.csproj

```diff
@@ -142,6 +142,7 @@
     <Compile Include="Models\ICommentModel.cs" />
     <Compile Include="Models\IInlineCommentModel.cs" />
     <Compile Include="Models\IPullRequestReviewCommentModel.cs" />
+    <Compile Include="Services\NullMetricsService.cs" Condition="$(Buildtype) != 'Internal'" />
```

Comment:
The check here should include `'` for safety (msbuild being all finicky about things at times. Like 

`Condition="'$(BuildType)' != 'Internal'"`